### ISSUE TITLE
Jetpack subscriptions: 'Notify me...' checkbox state influenced by 'Save my name...' checkbox

### ISSUE THREAD

**Issue by coder-karen** [Automattic/jetpack#19651]
#### Summary of issue

When using Jetpack's ~~comments and~~ subscription feature, if the `Save my name, email, and website in this browser for the next time I comment.` box is checked at any stage from a particular browser on a site, then the current checkbox state is also saved for `Notify me of new posts by email.`.

By default, every comment will then have that box checked.

<img width="718" alt="Screenshot 2021-04-26 at 13 35 55" src="https://user-images.githubusercontent.com/16754605/116083389-7730f580-a694-11eb-9168-b475d901f87d.png">

Is it possible for that particular checkbox to be ignored when the `Save my name, email, and website in this browser for the next time I comment.` is checked?

Mentioned in this forum thread: https://wordpress.org/support/topic/default-subscribe-to-blog-is-checked/

**Comment by jeherve** [comment]
I can't seem to reproduce the issue on my end. None of the checkboxes appear to be checked by default for me, and when I check one, it doesn't automatically check the other. Am I missing a step here?

**Comment by coder-karen** [comment]
Yes I haven't been clear in my testing steps. In fact I used quite a round-about flow and in going over it again realize it is a lot less clear than I explained it.

In order to view the 'Save my name' checkbox I did need to enabled Jetpack comments but assumed that was just a flow issue on my site.

Having then made sure to access a post as a logged out user, I then checked that box as well as clicking 'notify me of new comments' . That 'notify me' checkbox remained saved only if the Jetpack comments feature was then disabled. Clicking around a few other posts I noticed the 'notify me of comments' checkbox remained saved and details auto-filled. Clearing the browser cache cleared the selections. That was in Chrome.

However, I've tried to replicate again this morning and cannot, despite trying various different flows. I'd be happy to close this unless anyone can replicate it again.

**Comment by jeherve** [comment]
> That 'notify me' checkbox remained saved only if the Jetpack comments feature was then disabled. 

That's interesting. I cannot reproduce at the moment, but there may be something here. In Settings > Discussion, is the notify me option checked for posts?

**Comment by coder-karen** [comment]
It is, yes...

**Comment by jeherve** [comment]
Hm. That's odd. I still cannot reproduce, but I'll keep this issue open just in case it comes up again.

**Comment by jeherve** [comment]
Closing this for now because of the lack of activity on this. We can always reopen in the future if needed.
